doc/cli/tcpdump.md

```diff
@@ -0,0 +1,90 @@
+## Network Traffic Inspection
+
+`tcpdump` is useful tool for analyzing and diagnosing network problems.
+This document presents the limited feature set that exposed is in the
+CLI.  Administrator level users with shell access can use the full
+feature set, and is not described here.
+
+The following section is useful for understanding how to use the tool.
+A section called [Examples](#examples) follows that, which may be what
+you want to scroll down to.
+
+
+### Hardware Overview
+
+Using `tcpdump` effectively requires an understanding of how the
+underlying hardware works.  For a standard PC, or common SBC, the
```

Comment:
I'm not sure how widely used the term "Single Board Computer" is, maybe we should expand it?

Reply:
Will do, thanks!